docker-compose.yml

```diff
@@ -0,0 +1,30 @@
+version: '3'
```

Comment:
Right now a single file, but it could diverge into `docker-compose.yml` (locally) and `docker-compose.ci.yml` (for the `ci` stage).